src/scripts/achievements/AchievementHandler.ts

```diff
@@ -233,15 +233,16 @@ class AchievementHandler {
             }
             // Gyms
             GameConstants.RegionGyms[region]?.forEach(gym => {
-                AchievementHandler.addAchievement(`${gym} Gym regular`, 'Clear 10 times', new ClearGymRequirement(10, Statistics.getGymIndex(gym)), 1, region);
-                AchievementHandler.addAchievement(`${gym} Gym ruler`, 'Clear 100 times', new ClearGymRequirement( 100, Statistics.getGymIndex(gym)), 2, region);
-                AchievementHandler.addAchievement(`${gym} Gym owner`, 'Clear 1,000 times', new ClearGymRequirement(1000, Statistics.getGymIndex(gym)), 3, region);
+                const gymTitle: string = gym.includes('Elite') || gym.includes('Champion') ? gym : `${gym} Gym`;
+                AchievementHandler.addAchievement(`${gymTitle} Regular`, `Clear ${gymTitle} 10 times`, new ClearGymRequirement(10, Statistics.getGymIndex(gym)), 1, region);
+                AchievementHandler.addAchievement(`${gymTitle} Ruler`, `Clear ${gymTitle} 100 times`, new ClearGymRequirement( 100, Statistics.getGymIndex(gym)), 2, region);
+                AchievementHandler.addAchievement(`${gymTitle} Owner`, `Clear ${gymTitle} 1,000 times`, new ClearGymRequirement(1000, Statistics.getGymIndex(gym)), 3, region);
```

Comment:
Do we need these to be capitalized?
```suggestion
                AchievementHandler.addAchievement(`${gymTitle} regular`, `Clear ${gymTitle} 10 times`, new ClearGymRequirement(10, Statistics.getGymIndex(gym)), 1, region);
                AchievementHandler.addAchievement(`${gymTitle} ruler`, `Clear ${gymTitle} 100 times`, new ClearGymRequirement( 100, Statistics.getGymIndex(gym)), 2, region);
                AchievementHandler.addAchievement(`${gymTitle} owner`, `Clear ${gymTitle} 1,000 times`, new ClearGymRequirement(1000, Statistics.getGymIndex(gym)), 3, region);
```
As we are already also adding to old save file sizes with each of the Elite, Champions no longer including `Gym` in their title.

This isn't so much a problem for newer saves, but for saves that have a lot of achievements this could be quite a bit bigger.

![image](https://user-images.githubusercontent.com/7288322/95424462-27603a80-099f-11eb-9564-3d2f224a36e0.png)

_side note: it also looks like we went from `tourist` → `regular` at some point too 🤔, we should probably only be saving achievements that currently exist_

Reply:
Fair point - I thought it read better capitalized, but if it means we can put off switching the logic to use ids instead of names, I'll switch the spelling back.